### ISSUE TITLE
Python version requirements

### ISSUE THREAD

**Issue by larspalo** [nanoufo/copyright-hook#1]
Is the python 3.9 requirement really necessary, or could the version detection be better? On Ubuntu 20.04 3.8 is still the default and even if I have installed 3.9 from the repository this installation is not detected/used by the pre-commit hook. And, yes I did add a .bash-aliases file to change the python3 to point to my python3.9 executable which resulted in that python3 --version on the terminal would output something like 3.9.5.

**Comment by nanoufo** [comment]
Yes, Python 3.9 is required. The hook will crash on older Python versions.
To resolve the problem, you can override the Python version being used. Check out the instructions on how to do this at: https://pre-commit.com/#overriding-language-version.

**Comment by larspalo** [comment]
Ok, thanks, I'll try it the next time I try to commit anything.
